code/public/index.html

```diff
@@ -13,13 +13,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
-  </head>
+  <title>Technigo React App</title>
+</head>
 
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+<body>
+  <!-- <script src="../src/index.js"></script>-->
```

Comment:
Don't forget to remove commented out code.

You don't need to link the javascript - the dev server deals with that when you're working on your computer (after running `npm start`) and when you deploy, netlify runs `npm run build` which spits out a bundled project all nicely linked together.